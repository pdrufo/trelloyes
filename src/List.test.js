import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './List';

describe('List component', () => {
 //Smokescreen Test//
    it('renders withou crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<List key='a' header='myHeader' cards = {[1,2,3]} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

//Snapshot Test//
it('renders the UI as expected', () => {
    const tree = renderer
    .create(<List key='b'header='newHeader' cards={[]}/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
});

});