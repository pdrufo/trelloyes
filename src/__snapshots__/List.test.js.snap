// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component renders the UI as expected 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
      newHeader
    </h2>
  </header>
  <div
    class="List-cards"
  >
    <button
      className="List-add-button"
      type="button"
    >
      + Add Random Card
    </button>
  </div>
</section>
`;
